trucks.js

```diff
@@ -503,9 +518,138 @@ var foodTrucks = [
   }
 ];
 
-// this module should support the following methods:
-// getTrucks() - return all trucks
-// getTruck(name) - return the truck object matching 'name'
-// getFoodTypes() - return unique list of all associated food types (underscore has a function to help)
-// filterByDay(day) - return trucks with 'day' in schedule (use your filterByDay function from Module 3 homework)
-// filterByFoodType(foodType) - return trucks with associated 'foodType'
\ No newline at end of file
+/*************************************************************************
+* Description : getTrucks() - return all trucks
+* Parameters  : None
+* Return      : Array of trucks
+*************************************************************************/
+var getTrucks = function () {
+  var trucks = [];
+
+  foodTrucks.forEach(function(currentTruck) {
+    trucks.push(currentTruck);
```

Comment:
foodTrucks is already an array, so you don't need to put it into another array - instead just return it